catalog/core/catalog-core-metacardgroomerplugin/src/main/java/ddf/catalog/plugin/groomer/metacard/StandardMetacardGroomerPlugin.java

```diff
@@ -72,6 +73,10 @@ protected void applyCreatedOperationRules(CreateRequest createRequest, Metacard
             aMetacard.setAttribute(new AttributeImpl(Metacard.TAGS,
                     Collections.singletonList(Metacard.DEFAULT_TAG)));
         }
+
+        // upon creation of the metacard, generate attributes for created, modified dates
+        aMetacard.setAttribute(new AttributeImpl(ddf.catalog.data.types.Metacard.CREATED, now));
```

Comment:
I realize this is outside the scope of this change, but we should *_really_* change the name of this `Metacard` interface before it becomes too difficult to rename it.

Having `ddf.catalog.data.Metacard` and `ddf.catalog.data.types.Metacard` both existing is going to be confusing, especially as some of the consts defined in the former are _also_ defined in this new class.